cdist/conf/type/__acl/explorer/acl_is

```diff
@@ -19,5 +19,8 @@
 #
 
 if [ -e "/$__object_id" ]
-then getfacl "/$__object_id" | grep -E '^((default:|)(user|group)):[a-z]' || true
+then
+    getfacl "/$__object_id" \
+        | grep -E '(^((default:|)(user|group)):[a-z])|(^(default:|)mask::[a-z])' \
```

Comment:
after few months i see that i have been lazy.

i would go with `^((default:)?(user|group):[^:]|(default:)?mask::)`